lib/node_modules/@stdlib/stats/base/dists/gamma/mode/README.md

```diff
@@ -144,6 +144,101 @@ for ( i = 0; i < 10; i++ ) {
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dists/gamma/mean.h"
+```
+
+#### stdlib_base_dists_gamma_mean( alpha, beta )
+
+Returns the mean of a gamma distribution.
+
+```c
+double out = stdlib_base_dists_gamma_mean( 1.0, 1.0 );
+// returns 1.0
+```
+
+The function accepts the following arguments:
+
+-   **alpha**: `[in] double` shape parameter.
+-   **beta**: `[in] double` rate parameter.
+
+```c
+double stdlib_base_dists_gamma_mean( const double alpha, const double beta );
+```
```

Comment:
````suggestion
#include "stdlib/stats/base/dists/gamma/mode.h"
```

#### stdlib_base_dists_gamma_mode( alpha, beta )

Returns the mode of a gamma distribution.

```c
double out = stdlib_base_dists_gamma_mode( 1.0, 1.0 );
// returns 0.0
```

The function accepts the following arguments:

-   **alpha**: `[in] double` shape parameter.
-   **beta**: `[in] double` rate parameter.

```c
double stdlib_base_dists_gamma_mode( const double alpha, const double beta );
```
````